src/Symfony/Component/Messenger/Tests/Transport/RedisExt/ConnectionTest.php

```diff
@@ -42,13 +42,13 @@ public function testFromDsn()
     public function testFromDsnWithOptions()
     {
         $this->assertEquals(
-            new Connection(['stream' => 'queue', 'group' => 'group1', 'consumer' => 'consumer1', 'auto_setup' => false], [
+            new Connection(['stream' => 'queue', 'group' => 'group1', 'consumer' => 'consumer1', 'auto_setup' => false, 'xadd_maxlength' => 20000], [
                 'host' => 'localhost',
                 'port' => 6379,
             ], [
                 'serializer' => 2,
             ]),
-            Connection::fromDsn('redis://localhost/queue/group1/consumer1', ['serializer' => 2, 'auto_setup' => false])
+            Connection::fromDsn('redis://localhost/queue/group1/consumer1', ['serializer' => 2, 'auto_setup' => false, 'xadd_maxlength' => 20000])
```

Comment:
@Toflar should also be `stream_max_entries` I think?

Reply:
Of course, fixed sorry.